report-generator/src/test/java/com/teamscale/report/jacoco/JaCoCoXmlReportGeneratorTest.java

```diff
@@ -26,13 +26,21 @@
 	/** Ensures that the normal case runs without exceptions. */
 	@Test
 	void testNormalCaseThrowsNoException() throws Exception {
-		runGenerator("no-duplicates", EDuplicateClassFileBehavior.FAIL);
+		try {
+			runGenerator("no-duplicates", EDuplicateClassFileBehavior.FAIL);
+		} catch (EmptyReportException e) {
+			// We don't care about empty reports here
```

Comment:
when I read this, I'm unsure if either this can not happen during this test (then why do we catch it explicitly?) or it's what we expect to happen (then the comment is confusing for me). Can you please clarify this (also in the other tests in this file)?